src/pretix/plugins/stripe/templates/pretixplugins/stripe/checkout_payment_form_sepadirectdebit.html

```diff
@@ -0,0 +1,17 @@
+{% load i18n %}
+
+{% load bootstrap3 %}
+{% bootstrap_form form layout='horizontal' %}
+
+<div id="mandate-acceptance">
+{% blocktrans trimmed %}
+By providing your payment information and confirming this payment,
+you authorise to send instructions to your bank to debit your account
+and your bank to debit your account in accordance with those instructions.
+As part of your rights, you are entitled to a refund from your bank under
+the terms and conditions of your agreement with your bank.
+A refund must be claimed within 8 weeks starting from the date on which your account was debited.
+Your rights are explained in a statement that you can obtain from your bank.
+You agree to receive notifications for future debits up to 2 days before they occur.
+{% endblocktrans %}
+</div>
```

Comment:
Yes, I used the Stripe text and made it abstract, so it would work without the name.
If needed we can add the organizer name, however that may not be the SEPA mandate name, so not sure how to solve this.

Reply:
I think because of the legal implications of this, we should add a setting to configure the name.

I seem a little confused as to stripe not mentioning it, but doesn't a SEPA mandate also need to include the creditor ID? Or is that sufficient in the debit notification?

Reply:
So in the original Stripe SEPA mandate message, the name is included, however I made i abstract.
As for the creditor ID is entered into the Stripe account, no need to submit it.
I will see how we can add the SEPA mandate name so it shows up in the text.
I now only think that we actually also need translation for that, I will look into that for some of the languages I know